vaadin-grid-flow-integration-tests/src/main/java/com/vaadin/flow/component/treegrid/it/TreeGridBasicFeaturesPage.java

```diff
@@ -59,7 +59,7 @@ public TreeGridBasicFeaturesPage() {
         grid.setSelectionMode(SelectionMode.SINGLE);
         grid.setColumns("id", HierarchicalTestBean::toString,
                 Arrays.asList("id", "depth", "index"));
-        grid.setDataProvider(new LazyHierarchicalDataProvider(3, 2));
+        grid.setDataSource(new LazyHierarchicalDataProvider(3, 2));
```

Comment:
These should stay as `setDataProvider` as tree grid doesn't have a good DataView implementation and not using setDataSource in the test pages would give a better indication on that.

Reply:
Same for TreeGridDetachAttachPage, TreeGridRefreshAllPage and TreeGridScrollingPage.

Reply:
Done